components/ui/Card/ProductsCard.tsx

```diff
@@ -0,0 +1,165 @@
+import { Box, Text, Stack } from "@chakra-ui/react";
+import Image, { StaticImageData } from "next/image";
+import CustomButton from "../CustomButton/CustomButton";
+import { useMediaQuery } from "@chakra-ui/react";
+import moreIcon from "../../../public/icons/icon _more horiz circled outline_.png";
+
+interface CourseCardProps {
+  title: string;
+  price?: string;
+  Coachname: string;
+  description: string;
+  imageSrc: StaticImageData;

```

Comment:
ImageSrc implies that you should be passing a string, try to pass it as a string better than passing the whole image data